### DISCUSSION TITLE
Is early access for the playground or the entire models feature?

### DISCUSSION THREAD

**Discussion by ben-z** [community/community#134530] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm getting "unauthorized" errors when following instructions in "get started":
<img width="1331" alt="image" src="https://github.com/user-attachments/assets/600b5ae1-f1e8-4d65-b1e2-47202c25e257">

Is this because the waitlist is for the entire models feature, not only for playground?

**Comment by Akash1134** [comment, marked as answer]
Hi @ben-z ,

The early access is for all the models via Playground. Once you are approved, you should be able to use the listed models. 

The playground is in limited public beta and subject to change. To request access, [join the waitlist](https://github.com/marketplace/models/waitlist) if you haven't already.

Thanks!
